utils/grpc/health_check_test.go

```diff
@@ -0,0 +1,23 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"google.golang.org/grpc/health/grpc_health_v1"
+	"testing"
+	"time"
+)
+
+func TestHealthCheckClient(t *testing.T) {
+	client := HealthCheckClient("https://hub-dev.btfs.io")
```

Comment:
why only check hub? can you add a list of services to check? this may be useful foundation for monitoring all services later